source/uwp/Renderer/lib/AdaptiveElementParserRegistration.cpp

```diff
@@ -33,24 +33,22 @@ using namespace ABI::Windows::UI;
 
 namespace AdaptiveNamespace
 {
+    const char* c_upwElementParserRegistration = "447C3D76-CAAD-405F-B929-E3201F1537AB";
```

Comment:
>c_upwElementParserRegistration [](start = 16, length = 30)

should this be `c_uwpElementParserRegistration`?

Reply:
Sorry, i'm missing something here. Looks like it is that?

---
In reply to: [322488406](https://github.com/microsoft/AdaptiveCards/pull/3429#discussion_r322488406) [](ancestors = 322488406)

Reply:
`c_upw` vs. `c_uwp` :)

---
In reply to: [322490525](https://github.com/microsoft/AdaptiveCards/pull/3429#discussion_r322490525) [](ancestors = 322490525,322488406)

Reply:
Ha, i looked at it several times and didn't see that. Fixed!

---
In reply to: [322490525](https://github.com/microsoft/AdaptiveCards/pull/3429#discussion_r322490525) [](ancestors = 322490525,322488406)